### PR TITLE
Include coverage results for cc binaries called from sh_test in the Bazel coverage output file

### LINKED ISSUE #6257: Include coverage results for cc binaries called from sh_test in the Bazel coverage output file
Currently Bazel only outputs coverage results for java binaries called within a sh_test.

### PR DESCRIPTION
**Pull request by iirina**
Fixes #6257.

### CODE REVIEW

**Comment by iirina** [comment]
@mhlopko ping

File: tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator/Coverage.java [hunk 1]
```diff
@@ -66,13 +66,19 @@ static Coverage getOnlyTheseSources(Coverage coverage, Set<String> sourcesToKeep
...
-      if (sourcesToKeep.contains(source.sourceFileName())) {
+      if (!isCcSourceFile(source.sourceFileName())
+          || sourcesToKeep.contains(source.sourceFileName())) {
         finalCoverage.add(source);
       }
     }
     return finalCoverage;
   }
 
+  private static boolean isCcSourceFile(String filename) {
```
Comment (mhlopko) [inline tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator/Coverage.java]: Can you use CppFileTypes.C_SOURCE + CPP_SOURCE?

Comment (iirina) [reply tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator/Coverage.java]: That means packing an additional dependency into bazel_tools which makes the binary bigger. I copied those values to Constants.CC_EXTENSIONS and use it here to check if the file is a cc file.

**Comment by ulfjack** [comment]
This is in the process of being merged.

**Pull request closed without merging**